shared/src/commonMain/kotlin/whl/trending/ai/ui/hiring/HiringViewModel.kt

```
@@ -0,0 +1,133 @@
+                    HiringUiState.Ready(
+                        month = r.month,
+                        months = r.months,
+                        lastSyncedAt = r.lastSyncedAt,
+                        all = r.posts,
+                        // 进入页面一律无筛选态，且不记忆偏好——不替用户预设任何条件
+                        selected = emptyMap(),
+                    )
+                } else {
+                    HiringUiState.Unavailable
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)